<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MLinkClientTestClass</class>
 <widget class="QWidget" name="MLinkClientTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MLinkClientTest</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="rs232ComboBox"/>
     </item>
     <item>
      <widget class="QToolButton" name="rs232ConnectButton">
       <property name="icon">
        <iconset resource="MLinkClientTest.qrc">
         <normaloff>:/MLinkClientTest/icons/icons8-rs-232-female-filled-50.png</normaloff>:/MLinkClientTest/icons/icons8-rs-232-female-filled-50.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,0">
     <item>
      <widget class="QCheckBox" name="simplePacketEnableCheckBox">
       <property name="text">
        <string>Simple packet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="simplePacketLabel">
       <property name="minimumSize">
        <size>
         <width>57</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0,1,0">
     <item>
      <widget class="QCheckBox" name="ch0EnableCheckBox">
       <property name="text">
        <string>Ch0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="ch0DataSizeSpinBox">
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="value">
        <number>11520</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="ch0ProgressBar">
       <property name="value">
        <number>0</number>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="format">
        <string>Uploading %p%</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ch0Label">
       <property name="minimumSize">
        <size>
         <width>57</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>0/0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0,1,0">
     <item>
      <widget class="QCheckBox" name="ch1EnableCheckBox">
       <property name="text">
        <string>Ch1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="ch1DataSizeSpinBox">
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="value">
        <number>23040</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="ch1ProgressBar">
       <property name="value">
        <number>0</number>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="format">
        <string>Uploading %p%</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ch1Label">
       <property name="minimumSize">
        <size>
         <width>57</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>0/0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,0,0,0,1,0">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Ch2</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSpinBox" name="ch2DataSizeSpinBox">
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="value">
        <number>23040</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="ch2ProgressBar">
       <property name="value">
        <number>0</number>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="format">
        <string>Uploading %p%</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ch2Button">
       <property name="minimumSize">
        <size>
         <width>57</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>42</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Upload</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0,0,0,0,1,0">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Ch3</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSpinBox" name="ch3DataSizeSpinBox">
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="value">
        <number>23040</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="ch3ProgressBar">
       <property name="value">
        <number>0</number>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="format">
        <string>Downloading %p%</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ch3Button">
       <property name="minimumSize">
        <size>
         <width>57</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>42</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Download</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MLinkClientTest.qrc"/>
 </resources>
 <connections/>
</ui>
